pages/documentation/developer/repositories/pypowsybl.md

```diff
@@ -6,17 +6,23 @@ The PyPowSyBl project gives access to PowSyBl framework to Python developers. Th
 The available features are:  
 
 **Grid modelling**      
-- We can create an empty network ;
+- We can create an empty network.
```

Comment:
I know i was already there, but not sure we should advertise it here, it's the most useless feature for now because we cannot add elements !

Reply:
Indeed! removed.